terraform/aws/projects/nasa-veda.tfvars

```
@@ -63,7 +63,9 @@ hub_cloud_permissions = {
                     "arn:aws:s3:::sdap-dev-zarr",
                     "arn:aws:s3:::sdap-dev-zarr/*",
                     "arn:aws:s3:::usgs-landsat",
-                    "arn:aws:s3:::usgs-landsat/*"
+                    "arn:aws:s3:::usgs-landsat/*",
+                    "arn:aws:s3:::sentinel-cogs",
+                    "arn:aws:s3:::sentinel-cogs/*",
```

```suggestion
                    "arn:aws:s3:::sentinel-cogs/*"
```

I don't think the trailing comma is accepted here, but I'm not sure.

Also, we should update this for the staging hub as well if i recall correctly.

---

I need to focus on misc other things for the moment and won't finalize getting this done.

---

ahh yes. Thanks. I missed the trailing comma on the copy/paste. You can update for staging for consistency but it isn't needed at this time.